content/blog/2019/02/2019-02-04-running-jenkins-with-java-11.adoc

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: "Running Jenkins with Java 11"
```

Comment:
Needs more fanfares here, this is a GA announcement after all :)

Reply:
Agreed. Rewriting in this spirit.